### PR TITLE
add attribute custom_commands for user and group

### PR DESCRIPTION
**Pull request by safematix**
### Description

Add attribute custom_commands for user and group

Will create with a entry for user and/or group to run only a given command.

### CODE REVIEW

**Comment by safematix** [comment]
Hi,

please accept the pull request.

There is a problem with INSTANCE default-fedora-latestimage. (Error executing action `install` on resource 'yum_package[sudo]')

For the others everything looks very good. :smile:

Cheers,
Robert

**Comment by tas50** [comment]
@safematix Can you possibly add tests for this new functionality?

File: README.md [hunk 1]
```diff
@@ -50,6 +50,31 @@ To use attributes for defining sudoers, set the attributes above on the node (or
 }
 ```
 
+```json
+{
+  "default_attributes": {
+    "authorization": {
+      "sudo": {
+        "command_aliases": [
+          { name: 'TEST', command_list: ['/usr/bin/ls','/usr/bin/cat']},
```
Comment (thommay) [inline README.md]: this should all be valid json.

**Comment by safematix** [comment]
@thommay correct readme for command_aliases and custom_commands to valid json
@tas50 add spec check

**Pull request merged**